packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -115,8 +115,20 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 					}
 				}
 
+				const defaultDeniedFiles = [
+					".env",
+					".env.*",
+					"*.{crt,pem}",
+					"**/.git/**",
+				];
```

Comment:
Are these the default Vite ones? Did you have to copy them over?
Is it not possible to just add our `.dev.vars` etc paths over the top of what Vite provides??
Doesn't Vite just merge them automatically?

Reply:
They don't get merged automatically, unfortunately. I was expecting them to be exported for this purpose (like they export `defaultAllowedOrigins`) but they're not. I could open a PR to Vite to export.